### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by 4b0082**
## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [ ] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [x] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

File: So there's a plague on/sheet.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+{
+	"html": "So_theres_a_plague.html",
+	"css": "So_theres_a_plague.css",
+	"authors": "Indigo",
+	"preview": "So_theres_a_plague.png",
+	"instructions": "?????"
```
Comment (Anduh) [inline So there's a plague on/sheet.json:7]:
```suggestion
	"instructions": "A character sheet for 'So, there is a plague on' by Indigo. Rules available [here](https://4b0082.itch.io/buboes)"
```

Comment (Anduh) [reply So there's a plague on/sheet.json:7]: I'd suggest to include the at least this much info in this field, leaving question marks just makes it confusing. Look at any existing Roll20 sheet for comparison.

**Comment by oliviakristiansen** [comment]
What system is this for? I can't find anything about a So There's a Plague RPG online. [Rules must be readily available.](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) See the rules under "Minimum Requirements".

**Comment by Anduh** [comment]
@oliviakristiansen link to rules are linked in the sheet.json's instructions, should be enough.

**Comment by Anduh** [comment]
@oliviakristiansen link to rules are linked in the `sheet.json`'s "instructions"-section, should be enough.

**Comment by oliviakristiansen** [comment]
Could you please provide the System - is this an official sheet for So, theres a plague? Also, a `roll20userid` was not included.

File: So there's a plague on/sheet.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+{
+	"html": "So_theres_a_plague.html",
+	"css": "So_theres_a_plague.css",
+	"authors": "Indigo",
+	"preview": "So_theres_a_plague.png",
+	"instructions": "A character sheet for 'So, there is a plague on' by Indigo. Rules available [here](https://4b0082.itch.io/buboes)"
+}
```
Comment (4b0082) [inline So there's a plague on/sheet.json:8]: `"roll20userid": "857210"`

**Comment by 4b0082** [comment]
- Roll20 User ID added
- This is the official sheet

**Pull request merged**